Chinese/大话西游.md

```diff
@@ -37,3 +37,5 @@
 [周星驰](Zhou/Zhou.md) ?
 
 [小熊维尼](xinwang/xin.md)?
+
+[谢霆锋](TingfengXie/TingfengXie.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.